x-pack/plugins/ingest_manager/server/services/setup.ts

```diff
@@ -60,6 +65,53 @@ export async function setup(
   }
 }
 
+export async function setupFleet(
+  soClient: SavedObjectsClientContract,
+  callCluster: CallESAsCurrentUser
+) {
+  // Create fleet_enroll role
+  // This should be done directly in ES at some point
+  await callCluster('transport.request', {
+    method: 'PUT',
+    path: `/_security/role/${FLEET_ADMIN_ROLE}`,
+    body: {
+      cluster: ['monitor', 'manage_api_key'],
+      indices: [
+        {
+          names: ['logs-*', 'metrics-*', 'events-*'],
+          privileges: ['write', 'create_index'],
+        },
+      ],
+    },
+  });
+  const password = generateRandomPassword();
+  // Create fleet enroll user
+  await callCluster('transport.request', {
+    method: 'PUT',
+    path: `/_security/user/${FLEET_ADMIN_USERNAME}`,
+    body: {
+      password,
+      roles: [FLEET_ADMIN_ROLE],
+    },
+  });
+
+  // save fleet admin user
+  await outputService.updateOutput(soClient, await outputService.getDefaultOutputId(soClient), {
+    admin_username: FLEET_ADMIN_USERNAME,
+    admin_password: password,
```

Comment:
```suggestion
    fleet_enroll_password: password,
```
I think it is important to also change the naming in the code to not have confusion.